frontend/web/components/ExternalResourcesTable.tsx

```diff
@@ -30,6 +47,177 @@ const ExternalResourcesTable: FC<ExternalResourcesTableType> = ({
       toast('External resources was deleted')
     }
   }, [isDeleted])
+  return (
+    <Row className='list-item' key={externalResource?.id}>
+      <Flex className='table-column mt-1'>
+        <Row className='font-weight-medium'>
+          {externalResource?.type === 'GITHUB_ISSUE'
+            ? Constants.githubType.githubIssue
+            : Constants.githubType.githubPR}
+          <Button
+            theme='text'
+            href={`${externalResource?.url}`}
+            target='_blank'
+            className='fw-normal ml-1 mt-1'
+          >
+            <Tooltip
+              title={
+                <Row>
+                  {`#${externalResource?.url.replace(/\D/g, '')}`}{' '}
+                  <div className='ml-1 mb-1'>
+                    <Icon name='open-external-link' width={14} fill='#6837fc' />
+                  </div>
+                </Row>
+              }
+              place='right'
+            >
+              {`${externalResource?.url}`}
+            </Tooltip>
+          </Button>
+        </Row>
+      </Flex>
+      <div className='table-column text-center' style={{ width: '80px' }}>
+        <div className='font-weight-medium mb-1'>
+          {externalResource?.metadata?.status}
+        </div>
+      </div>
+      <div className='table-column text-center' style={{ width: '80px' }}>
+        <Button
+          onClick={() => {
+            deleteExternalResource({
+              external_resource_id: `${externalResource?.id}`,
+              feature_id: featureId,
+              project_id: projectId,
+            })
+          }}
+          className='btn btn-with-icon'
+        >
+          <Icon name='trash-2' width={20} fill='#656D7B' />
+        </Button>
+      </div>
+    </Row>
+  )
+}
+
+const PermanentRow: FC<PermanentRowType> = ({
```

Comment:
The term PermanentRow is confusing, it's not obvious what this is

Reply:
Renamed